docs/PRIVACY_POLICY.md

```diff
@@ -0,0 +1,538 @@
+
+- To facilitate account creation and authentication and otherwise manage user accounts. We may process your information so you can create and log in to your account, as well as keep your account in working order.
+
+- To deliver and facilitate delivery of services to the user. We may process your information to provide you with the requested service.
+
+- To respond to user inquiries/offer support to users. We may process your information to respond to your inquiries and solve any potential issues you might have with the requested service.
+
+- To send administrative information to you. We may process your information to send you details about our products and services, changes to our terms and policies, and other similar information.
+
+- To fulfill and manage your orders. We may process your information to fulfill and manage your orders, payments, returns, and exchanges made through the Services.
+
+- To request feedback. We may process your information when necessary to request feedback and to contact you about your use of our Services.
+
+- To send you marketing and promotional communications. We may process the personal information you send to us for our marketing purposes, if this is in accordance with your marketing preferences. You can opt out of our marketing emails at any time. For more information, see "[WHAT ARE YOUR PRIVACY RIGHTS?](#11-what-are-your-privacy-rights)" below.
+
+- To protect our Services. We may process your information as part of our efforts to keep our Services safe and secure, including fraud monitoring and prevention.
+
+- To identify usage trends. We may process information about how you use our Services to better understand how they are being used so we can improve them.
+
+- To save or protect an individual's vital interest. We may process your information when necessary to save or protect an individual's vital interest, such as to prevent harm.
+
+### 3. WHAT LEGAL BASES DO WE RELY ON TO PROCESS YOUR INFORMATION?
+
+_In Short: We only process your personal information when we believe it is necessary and we have a valid legal reason (i.e., legal basis) to do so under applicable law, like with your consent, to comply with laws, to provide you with services to enter into or fulfill our contractual obligations, to protect your rights, or to fulfill our legitimate business interests._
+
+_If you are located in the EU or UK, this section applies to you._
+
+The General Data Protection Regulation (GDPR) and UK GDPR require us to explain the valid legal bases we rely on in order to process your personal information. As such, we may rely on the following legal bases to process your personal information:
+
+- Consent. We may process your information if you have given us permission (i.e., consent) to use your personal information for a specific purpose. You can withdraw your consent at any time. Learn more about [withdrawing your consent](#16-how-can-you-contact-us-about-this-notice).
```

Comment:
i couldnt find a section that talks about "withdrawing your consent". so, i linked to "contact-us" section.
@isbo fyi!